src/util/message.h

```diff
@@ -232,6 +238,12 @@ class messaget
       return func(*this);
     }
 
+    json_stream_arrayt &json_stream()
```

Comment:
doxygen

Reply:
Why is there a `json_stream_arrayt` variant, but no `json_stream_objectt` one?